src/test/java/net/coru/kloadgen/input/avro/AvroSubjectPropertyEditorTest.java

```diff
@@ -44,9 +52,44 @@ public void iterationStart(@Wiremock WireMockServer server) throws IllegalAccess
     JMeterContextService.getContext().getProperties().put(SCHEMA_REGISTRY_USERNAME_KEY, "foo");
     JMeterContextService.getContext().getProperties().put(SCHEMA_REGISTRY_PASSWORD_KEY, "foo");
 
-    AvroSerializedConfigElement avroSerializedConfigElement = new AvroSerializedConfigElement("AvroSubject", Collections.emptyList(), null);
+    AvroSerializedConfigElement avroSerializedConfigElement = new AvroSerializedConfigElement("avroSubject", Collections.emptyList(), null);
     JMeterVariables variables = JMeterContextService.getContext().getVariables();
     avroSerializedConfigElement.iterationStart(null);
     assertThat(variables).isNotNull();
+    assertThat(variables.get("SampleEntity")).isNotNull();
+  }
+
+
+  private static Stream<Arguments> parametersForMergeValue() {
+    return Stream.of(Arguments.of(new ArrayList<FieldValueMapping>(), new ArrayList<FieldValueMapping>(), new ArrayList<FieldValueMapping>()),
+        Arguments.of(new ArrayList<FieldValueMapping>(Arrays.asList(new FieldValueMapping("fieldName", "fieldType"))),
+            new ArrayList<FieldValueMapping>(),
+            new ArrayList<FieldValueMapping>()),
+        Arguments.of(new ArrayList<FieldValueMapping>(Arrays.asList(new FieldValueMapping("fieldName", "fieldType"))),
+            Arrays.asList(new FieldValueMapping("fieldSchema1", "string")),
+            Arrays.asList(new FieldValueMapping("fieldSchema1", "string"))),
+        Arguments.of(new ArrayList<FieldValueMapping>(Arrays.asList(new FieldValueMapping("fieldSchema1", "int"))),
+            Arrays.asList(new FieldValueMapping("fieldSchema1", "string")),
+            Arrays.asList(new FieldValueMapping("fieldSchema1", "string"))),
+        Arguments.of(new ArrayList<FieldValueMapping>(Arrays.asList(new FieldValueMapping("fieldSchema1", "string"))),
+            Arrays.asList(new FieldValueMapping("fieldSchema1", "string")),
+            Arrays.asList(new FieldValueMapping("fieldSchema1", "string"))),
+        Arguments.of(new ArrayList<FieldValueMapping>(Arrays.asList(new FieldValueMapping("fieldSchema1", "string", 0, "[\"value1\"]"))),
+            Arrays.asList(new FieldValueMapping("fieldSchema1", "string"), new FieldValueMapping("field2", "string")),
+            Arrays.asList(new FieldValueMapping("fieldSchema1", "string", 0, "[\"value1\"]"), new FieldValueMapping("field2", "string"))),
+        Arguments.of("value", Arrays.asList(new FieldValueMapping("field2", "string")), Arrays.asList(new FieldValueMapping("field2", "string"))));
   }
+
+  @ParameterizedTest
+  @MethodSource("parametersForMergeValue")
+  public void mergeValueTest(Object atributeListTable, List<FieldValueMapping> attributeList, List<FieldValueMapping> expected) {
+
+    List<FieldValueMapping> result = new AvroSubjectPropertyEditor().mergeValue(atributeListTable, attributeList);
+
+    assertEquals(expected, result);
```

Comment:
We are using assertJ for tests assertions, take a look and update your code, please.